install.sh

```diff
@@ -2,10 +2,10 @@
 set -e
 source prerequisites.sh
 
-if [ ! -e "venv/bin/python2.7" ]; then
-    echo "could not find venv/bin/python2.7, recreating venv"
+if [ ! -e "venv/bin/python3.5" ]; then
```

Comment:
could consider python3.6?

Reply:
Alas, we don't have it available on Ubuntu LTS in an easy way

Reply:
By Ubuntu LTS you mean an old LTS? It should be easily available on the latest LTS.

Reply:
16.04 is the current one we use, [18.04 has some issues] to be solved before we can use it.